### DISCUSSION TITLE
Welcome to Github Discussions on Ledger Live Desktop

### DISCUSSION THREAD

**Discussion by cthepot** [LedgerHQ/ledger-live-desktop#3406] [General]
## :rocket: Welcome to Github Discussions on Ledger Live Desktop

Feel free to share your thoughts and feedback on Ledger Live Desktop.

Discussions is a place for:
- [x] Feature requests
- [x] UX/UI suggestions
- [x] Questions about features

For :spider::bug: bugs, feel free to report them directly in the issues tab.

**Comment by djspops** [comment]
Ledger Nano S -- new device out to the box not working?
Version 2.20.0 broken on Win10  -

“Welcome to Ledger Nano S” --- neither button works

**Reply by cthepot** [reply]
hi @djspops, could you clarify what error messages or which step does not work on Live version 2.20? 
For Ledger Hardware (Nano S) issues you can contact customer support [here](https://support.ledger.com/hc/en-us).

**Comment by bkdddye5455** [comment]
Trouble shoot....why can't I restore my Nano S.  I've entered everything and it liked it all but still nothing  I'm really worried now.
